lib/surface/components/link.ex

```diff
@@ -27,37 +28,68 @@ defmodule Surface.Components.Link do
 
   use Surface.Component
 
-  import Phoenix.HTML.Link, only: [link: 2]
-
-  @doc "Place to link to"
+  @doc "The page to link to"
   prop to, :string, required: true
 
+  @doc "The method to use with the link"
+  prop method, :atom, default: :get
```

Comment:
Does this bring https://github.com/msaraiva/surface/pull/242 back?

Reply:
Couldn't reproduce #242

If the value of `method` is `:get` (default or explicitly set) it won't gat added as an attribute to the \<a> tag, see line  85. This was inspired by Phoenix's `link/2`.